src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -115,11 +115,17 @@ public static double Abs(Complex value)
                 return double.PositiveInfinity;
             }
 
+            return Hypot(value._real, value._imaginary);
+        }
+
+        private static double Hypot (double a, double b)
+        {
             // |value| == sqrt(a^2 + b^2)
```

Comment:
Yes, this is a good point. Doesn't `Sqrt(+inf, +inf)` still return `(NaN, NaN)` as it did before, rather than `(+inf, +inf)` as it should?